### ISSUE TITLE
Unit Stats of lvl3-4 Elves in 1.18 Are Unreasonable

### ISSUE THREAD

**Issue by ForestDragon-wesnoth** [wesnoth/wesnoth#8741]
### Describe the desired feature

I've noticed recently that stats of elvish high level units were buffed beyond any reasonable amount. Shyde got 14-3 magic ranged instead of 8-3, Sylph got 16-5 magic ranged instead of 10-5, Sharpshoot got 12-5 instead of 10-5. These stats are completely unreasonable on multiple levels.

1. Elvish level-ups were already often above average compared to similar units in other factions. But now they completely outshine many of them. Compare Sharpshooter's 12-5 with Huntsman's 9-4. The poacher line was always somewhat behind but now they are really way behind, that 12-5 feels like 4 lvl damage. Sylph's 16-5 surpasses feared Ancient Lich's 13-5 (and Ancient Lich is considered very strong lvl 4). It now feels unfair to get 24 xp for killing both Huntsman and Elvish Sharpshooter/Shyde, considering how much stronger the elvish 3 lvls are

2. Arcane rework was supposed to make it easier for undead, but Sylph got buffed so much that even with new arcane resistance of undead her damage against undead became higher than it was before rework. And combine it with increased damage against pretty much everything else. Elvish Sylph was already a very strong 4 lvl that could win you games, and she received insane buff on top of that.

3. Elvish Shyde may be an even more outrageous case. She's lvl 3, she has now 14-3 ranged pierce damage, with magic. 42 damage. Let's compared with other lvl 3 ranged-focused units: Dragonguard is 40-1, Master Bowman has 11-4 as well as Elvish Avenger, who originally had 10-4 in 1.16. Orcish Slurbow has 12-3. Banebow has 14-3, literally same damage, with no magic. Elvish Shyde has ranged damage comparable to all lvl 3 ranged-focused damage dealing units, and in addition to that she also has healing, flying, 6 movepoints, good terrain defence and her attack is magic. She has literally a full pack, a regiment of Shydes and you don't really need other units

4. Increased prices and xp are not a viable way to balance op lvl3-4s whatsoever because you almost never buy 3 lvl units and xp to get them is usually abundant

5. Another side effect is that elves pay 3 upkeep for units of 4 lvl power (and 4 upkeep for 5 lvl worthy Sylph). Another unreasonable advantage of Elves over the others

I discussed the issue with my friends and propose the following changes in 1.19 to improve the situation 
Enchantress - faerie fire 13-4 > 11-4, entangle 7-4 > 6-4, xp 198 > 180, price 70 > 62
Sylph: faerie fire 16-5 > 13-5 price 161 > 135
Sharpshooter: longbow 12-5 > 10-5 price reduced later
Avenger: bow 11-4 > 10-4, price 66 > 60
Shyde: thorns 14-3 > 10-3, ensnare 8-3 > 7-3, price 69 > 58

**Comment by soliton-** [comment]
Also see #8740.

**Comment by kabachuha** [comment]
The fact that the upgradable (possible to advance from Sorceress) Sylph is stronger than the Ancient Lich (who is often supposed to be a boss, and not obtainable without a macro definition) is absurd

**Comment by babaissarkar** [comment]
Sylph was earlier just 10-5 which got buffed to 16-5, the most dramatic one, not consider that faerie fire is already magical. This is also going to break a lot of UMC's balancing.

**Comment by igorbat** [comment]
Another important thing here is following: this change happens to be very unexpected by community: both modders and players. Wesnoth possibly needs a mechanism to prevent such an issue in advance.  Clean communication at forums/discord (aka drawing **additional** attention) for such a big changes would be highly welcomed.

**Comment by CelticMinstrel** [comment]
That's _technically_ possible, but not really plausible for a great many reasons.

**Comment by Discontinuum** [comment]
I completely agree with arguments presented in this issue and in #8740 These buffs are completely mad. These elvish units were already semi-op: you can complete Heir to the Throne just spamming Shammies and advancing them to Shydes and Sylph, this challenge was done many times, I saw on the forums. Elves were by no means weak, especially the shaman line.

Moreover, addons like "Legend of the Invincibles" and "Reign of the Lords" add extra levels 4-5 for all those units, and new buffed elves are stronger than those extra advancements! Really feels the current elves have to be +1 level for it to be fair.

So I personally would support a proposal in #8740 to revert this buffs of elves entirely 

And yes, this sweeping change was made completely silently from the community. A github PR, nobody noticed it, merged. Such serious changes at least should undergo a public discussion. Voting may he excessive, it has its problems, but discussions with active players and mod makers are really needed for such cases

**Comment by Hejnewar** [comment]
> And yes, this sweeping change was made completely silently from the community. A github PR, nobody noticed it, merged.

[link](https://forums.wesnoth.org/viewtopic.php?p=681418#p681418)

At least dont lie. It makes you look bad you know?

**Comment by Discontinuum** [comment]
A long list of lines with sweeping changes, with no argumentation, hidden under the spoiler on the 3rd page of the topic with mostly small balancing changes is not a wide public discussion

**Comment by ForestDragon-wesnoth** [comment]
I agree with Discontinuum

**Comment by Mirion147** [comment]
I would just like to ask that these "multiplayer changes" be addressed in a place people who don't take part in multiplayer can see, since they clearly impact campaigns and single player experience as well.

**Comment by Hejnewar** [comment]
> A long list of lines with sweeping changes, with no argumentation, hidden under the spoiler on the 3rd page of the topic with mostly small balancing changes is not a wide public discussion

 And yet still there are 3,5 more pages of comments aware of that. Im not gonna care about people who only wake up after MONTHS because it was discussed for months, on forums, in PR, on discord, and even on mp server I literally was going and asking people for opinions. If you arent involved in the community enough, or didnt notice thats on you. In not gonna care. I did enough.  
 
 > I would just like to ask that these "multiplayer changes" be addressed in a place people who don't take part in multiplayer can see, since they clearly impact campaigns and single player experience as well.
 
 If there is better place then sure. Point me to a place where I would also need to post them and Im fine with that, but till now that was the place where changes were always discussed.

**Comment by Mirion147** [comment]
>  If there is better place then sure. Point me to a place where I would also need to post them and Im fine with that, but till now that was the place where changes were always discussed. 

- I would suggest posting in #content-promo about unit changes and not listing them as mp changes, since they're simply unit changes and effect all sides of wesnoth.

- In addition I think it would be a good idea to have an add-on on the stable version of Wesnoth where players can access these changes in game without having to switch over to the  development version (obviously for instances where the stable won't support a change that won't be available, but it would be in the add-on description).

- Also, at the very least, post in faction and era development, possibly with posts in both mp and campaign development where they link to the post. But I'll be honest, if you're actually looking for a response and conversation about it, discord seems the more likely place to go imo.

This brings into question why there aren't better places to discuss topics on the official discord, but I won't go more into that topic here.

**Comment by Discontinuum** [comment]
> If you arent involved in the community enough

You can't apply this to me because you should know my opinion about all those changes, we talked with you a lot in different servers. Many people now supporting the issue are also active in the community, they make mods, play, post in discords, on the forums. If it's a surprise to you that many active people are against those changes, maybe you weren't really asking them? 

And overall, it doesn't matter to the point of the issue when people discovered that the changes are insane, after months, or after hours. These changes are still insane and should be reverted imho

**Comment by Hejnewar** [comment]
> I would suggest posting in #content-promo about unit changes and not listing them as mp changes, since they're simply unit changes and effect all sides of wesnoth.

Sure. 

> In addition I think it would be a good idea to have an add-on on the stable version of Wesnoth where players can access these changes in game without having to switch over to the development version (obviously for instances where the stable won't support a change that won't be available, but it would be in the add-on description).

Thats harder to do. Depends on time and possibility. So 50/50 maybe it will maybe it wont. This wouldnt affect campaigns anyway.

> Also, at the very least, post in faction and era development, possibly with posts in both mp and campaign development where they link to the post. But I'll be honest, if you're actually looking for a response and conversation about it, discord seems the more likely place to go imo.

Also sure. 

> You can't apply this to me because you should know my opinion about all those changes, we talked with you a lot in different servers.

You just said they were silent to you. And Im not a mind reader. Maybe try communicating instead of assuming that people should know what you think?

**Comment by Discontinuum** [comment]
> You just said they were silent to you. And Im not a mind reader. Maybe try communicating instead of assuming that people should know what you think?

Whatever it was, it's offtopic and doesn't relate to the issue. And the issue is that the changes are mad

**Comment by ForestDragon-wesnoth** [comment]
Yeah, the changes are a complete mockery of wesnoth's balance

**Comment by Hejnewar** [comment]
Then see you in a year or a half whenever I actually start working on that again. Until then have fun.

**Comment by AnekronCZ** [comment]
I very much agree with Mirion that labelling these changes as MP was unfortunate. I at first thought that the focus would be on general balance, which mostly touches lvl 1 and lvl 2 units.

I also feel that the current Sylph loses what made the Shaman line unique in comparison to the loy Mage line. The Great Mage is now strictly weaker than the Sylph with less attacks, worse movement and lower HP. My assumption always was that while the elves are more magically gifted than humans, they also don't use their power much for destruction and prefer to focus on other areas (hence healing + slow). These differences added to the unique faction identity for me.

**Comment by ForestDragon-wesnoth** [comment]
Yeah, Anekron is right, Sylph having 16-5 while Great Mage has 17-4 and Enchantress having 13-4 compared to Archmage's 12-4 is just laughable

**Comment by babaissarkar** [comment]
Sylph also has much higher mobility since she can fly, compared to Great Mage.

**Comment by ForestDragon-wesnoth** [comment]
> Sylph also has much higher mobility since she can fly, compared to Great Mage.
also she has a slowing attack and a melee attack with same total damage (6x3 or 9x2 = 18) but magical special, making her even more versatile

**Comment by CelticMinstrel** [comment]
>     * I would suggest posting in #content-promo about unit changes and not listing them as mp changes, since they're simply unit changes and effect all sides of wesnoth.

Hiding something like this behind a login wall sounds like a terrible, _terrible_ idea. If such a thing is going to be posted, it should be a public, anonymously-accessible place, such as the forums.

That said, posting a _link_ in #content-promo would be fine.

**Comment by ForestDragon-wesnoth** [comment]
> > * I would suggest posting in #content-promo about unit changes and not listing them as mp changes, since they're simply unit changes and effect all sides of wesnoth.
> 
> Hiding something like this behind a login wall sounds like a terrible, _terrible_ idea. If such a thing is going to be posted, it should be a public, anonymously-accessible place, such as the forums.
> 
> That said, posting a _link_ in #content-promo would be fine.

the idea is posting info about major balance discussions in multiple places, to reach as wide a public as reasonably possible

**Comment by CelticMinstrel** [comment]
It's fine to get some feedback from Discord, but the fact remains that Discord is a private room, whereas the reasoning for decisions should be placed out in the open. Even for people who rarely post on the forums, I would think seeing a link to an important discussion on the forum would be reason enough to log in and make a post if they care about the topic; and for those who can't be bothered and prefer to talk only on Discord, it would be ideal for someone else to summarize their arguments if they seem relevant to the eventual decision. (That someone would probably be the person _making_ the decision.)

**Comment by Mirion147** [comment]
> It's fine to get some feedback from Discord, but the fact remains that Discord is a private room, whereas the reasoning for decisions should be placed out in the open. Even for people who rarely post on the forums, I would think seeing a link to an important discussion on the forum would be reason enough to log in and make a post if they care about the topic; and for those who can't be bothered and prefer to talk only on Discord, it would be ideal for someone else to summarize their arguments if they seem relevant to the eventual decision. (That someone would probably be the person _making_ the decision.)

This sounds like the job of the person who wants the feedback. If you actually care about feedback you'll get it where you can. If you want to save it in the forum, that's up to you. If you want to listen to it just cause it was posted in discord, that's up to you. If you want to ignore it because they didn't take the time to log into a website they only visit once a month, if ever (as opposed to discord that they check every day) then that's up to you. It's up to the presenter that is looking for feedback.

At the end of the day its up to the person working on it. But to just say "I posted these stats under a rug in the basement of a derelict building 400 miles away from you, it's not my fault you didn't go find it" (not an actual quote) sounds like you don't actually want feedback and just want things your way (you being a general you) 

Please note that I'm not upset or meaning to be rude at all. But I don't understand why we're acting like the forums get the traction they used to get. Things should be posted there, yes, but that doesn't mean just cause you only post on discord that your opinions don't matter.

**Comment by CelticMinstrel** [comment]
> This sounds like the job of the person who wants the feedback. If you actually care about feedback you'll get it where you can. If you want to save it in the forum, that's up to you. If you want to listen to it just cause it was posted in discord, that's up to you. If you want to ignore it because they didn't take the time to log into a website they only visit once a month, if ever (as opposed to discord that they check every day) then that's up to you. It's up to the presenter that is looking for feedback.

Partly, but not entirely. If there's something like a poll then that should be on the forums as well, so that anyone can vote.

**Comment by Pentarctagon** [comment]
Probably worth mentioning that the announcements for both 1.17.19 and 1.17.20 do explicitly mention that there were major changes to the Default era units. The Discord announcement also did link (at the time, seems the header sub-links don't work when you add to the beginning of the file instead of the end) to the list of all the changes in the changelog.

Could the links to the changelog been done better/more consistently? Sure. But it's not like the information was secret or even particularly hidden.

**Comment by demario-w** [comment]
> Probably worth mentioning that the announcements for both 1.17.19 and 1.17.20 do explicitly mention that there were major changes to the Default era units. The Discord announcement for 1.17.20 also did link (at the time, seems the header sub-links don't work when you add to the beginning of the file instead of the end) to the list of all the unit stat changes in the changelog.
> 
> Could the links to the changelog been done better/more consistently? Sure. But it's not like the information was secret or even at all hidden.

The elephant in the room is that this team of developers constantly remind everyone that only devs know and the players feedback will be generally [mocked](https://github.com/wesnoth/wesnoth/issues/6971#issuecomment-1221186649) and discarded (esp. on github). That is the culture enshrined in this project at this time.

The MP developer in question is also repeatedly claiming that he represents the MP community inside the project and is constantly [in contact](https://github.com/wesnoth/wesnoth/pull/8688#issuecomment-2037999686) with MP players. It tends to make people think there are closed venues where he discussed this changes with MP players.
From the reaction after the release, it appears it was misleading and he just made up the numbers on his own.

The only way to have these changes to be usefully reviewed beforehand was to publish them as an add-on available on stable branch.

**Comment by Pentarctagon** [comment]
@demario-w Except... that doesn't actually happen. You seem to instinctively dislike basically *anything* we do though, so I guess I'm not surprised by the comment.

**Comment by nemaara** [comment]
@ForestDragon-wesnoth There is too much for me to read quickly but I am okay with your proposed changes as far as SP goes. Other ideas is raise exp more or lower HP, I am also okay with these. I think as long as we don't lower exp too much campaigns will be fine. 

Since we should keep stats the same between SP and MP any proposed changes also need Hejnewar's approval.

**Comment by babaissarkar** [comment]
Look at how many of lvl 4s are locked by default (https://www.wesnoth.org/macro-reference.html#file:optional_unit_advancements.cfg), I think creating a generic LVL_4_LOCK macro might be useful.
Though I'm not really familliar with the logic why those particular units are locked.

**Comment by ForestDragon-wesnoth** [comment]
> Look at how many of lvl 4s are locked by default (https://www.wesnoth.org/macro-reference.html#file:optional_unit_advancements.cfg), I think creating a generic LVL_4_LOCK macro might be useful. Though I'm not really familliar with the logic why those particular units are locked.

and 1.18 sylph deals more damage than both the locked Ancient Lich and locked Armageddon Drake

**Comment by demario-w** [comment]
> @demario-w Except... 

I will not derail this discussion further. I just wanted to call out your usual technique of spreading the responsibility of the situation to everyone who stayed silent. When you devs use hostile and oppressive ways to disregard the opinion of community members, you lose the ability to claim we're all on the same boat and you must take the full responsibility of the changes you made.

**Comment by loonycyborg** [comment]
There are so many ways to approach same problem that it's impossible to avoid disregarding someone. Nonetheless it's obvious to me that maintainers are open to compromise, I don't see a point in making a Fronde from this.

**Comment by Hejnewar** [comment]
> The elephant in the room is that this team of developers constantly reminds everyone that only devs know and the players feedback will be generally https://github.com/wesnoth/wesnoth/issues/6971#issuecomment-1221186649 and discarded (esp. on github). That is the culture enshrined in this project at this time.

It is so mocked that I actually opened subsequent issues for some of the stuff and also fixed some of the things form that issue (whatever I had the skill for) and pushing for other like the turn bell (and I wasnt even alone on this). But thanks I think I see 1 more thing that I could improve with my skill now.

But considering you history of trying to skill shame people and use other techniques (like above not checking the facts, or just doing that malicious in the worse case) to get what YOU want im not at all surprised. And honestly because of that I have no reason to ever take your feedback seriously. You could actually take a lesson from Mirion if you want to give some useful feedback (which I dont think you want but anyway) and have something happen.

**Comment by Discontinuum** [comment]
While @demario-w 's text may be excessively harsh, I would like to point out that the development team definitely has problems with connecting with the multiplayer community and listening to their opinions.

As far as I understand, Hejnewar is the one assigned for multiplayer content and events, but he lacks experience in arranging large scale events, so he can't know all the challenges such events pose. As we see in this and Dalas' issue, he's reluctant to listen to criticism of other players, modmakers and fellow devs. He's also very busy, as he noted earlier in this topic he would work on this issue only in a year. So I think the development team of Wesnoth definitely needs more help of volunteers from the MP community with the MP aspect of Wesnoth

**Comment by Hejnewar** [comment]
> So I think the development team of Wesnoth definitely needs more help of volunteers from the MP community with the MP aspect of Wesnoth

Sure, if for event making please message me Im open to ideas and suggestions I can also provide the support necessary in form of eras or advertisement as I recently did for the 2v2 tournament (and as far as i know people actually joined thanks to that so im happy) maybe even reward. 

> As we see in this and Dalas' issue, he's reluctant to listen to criticism of other players, modmakers and fellow devs. He's also very busy, as he noted earlier in this topic he would work on this issue only in a year.

Where? I said there that whatever Yumi decides (I guess I could expand it to community) I will balance, but yeah I would like Yumi to agree first. But now I think I will just give balance to Dalas and slander all the changes together with the community in the futer to grow closer to it again. This also solves the time issue, but I dont plant to make any balance PR until I have something more to do because realistically for 1.18 nothing will change anyway so I dont se any urgency. 

So yeah I guess you should read more carefully what I write because you are again at best inaccurate and at worst just trying to slander me, which makes me less likely to listen to you opinion at all.

**Comment by Mirion147** [comment]
> @ForestDragon-wesnoth There is too much for me to read quickly but I am okay with your proposed changes as far as SP goes. Other ideas is raise exp more or lower HP, I am also okay with these. I think as long as we don't lower exp too much campaigns will be fine. 
> 
> Since we should keep stats the same between SP and MP any proposed changes also need Hejnewar's approval. 

Not to cut the bickering short, but I'm guessing we can close this topic since Yumi already agreed to the changes. I don't think anyone is specifically looking to fight, so let's just drop it and move on. Thank you for taking the time to discuss the topic and being open to changes, and thank you for your approval of the changes (both of you). Have a good day, we look forward to seeing these units get updated so we can all get back to enjoying wesnoth!

**Comment by ForestDragon-wesnoth** [comment]
opened a PR here for this issue - https://github.com/wesnoth/wesnoth/pull/8749

**Comment by Pentarctagon** [comment]
I think the remaining question is what more could have been done to make people aware of these changes. The fact that there were significant changes to the Default era factions was mentioned in two release announcements on the forums and on Discord (and on Fosstodon and itch.io for that matter), the Reddit release announcements link back to the forum announcements, it was discussed on Discord further, on the forums, and in the individual PR here on github for the Rebels' changes.

That's... everywhere we have. There are no more places it could've been announced or discussed and all of them are public (yes, I know Discord technically isn't).

**Comment by kabachuha** [comment]
@Pentarctagon How about making a mechanism for **in-game** main menu announcement pop-up windows like in games such as Hearts of Iron, Cities Skylines and so on?

**Comment by Mirion147** [comment]
> I think the remaining question is what more could have been done to make people aware of these changes. The fact that there were significant changes to the Default era factions was mentioned in two release announcements on the forums and on Discord (and on Fosstodon and itch.io for that matter), the Reddit release announcements link back to the forum announcements, it was discussed on Discord further, on the forums, and in the individual PR here on github for the Rebels' changes.
> 
> That's... everywhere we have. There are no more places it could've been announced or discussed and all of them are public (yes, I know Discord technically isn't).

I think part of it, for me, is just my lack of understanding. I didn't realize that by changing these eras it would impact single player. It makes perfect sense when I put thought to it, but I never have before. So it could be that other people did the same thing. Either way, for my part I would like to apologize about simply not understanding the lingo (which is easy to understand).

I do also think that having these changes be at least somewhat playable on the stable version would go a long way to allowing players and umc creators to get a feel for how things would go. (Preferably with an input in the file that a umc creator could place their campaign, or whichever campaign they want into and allow that campaign to run with the updated changes)

**Comment by Dalas121** [comment]
It's been a while so my memory is fuzzy, but this is my best recollection:

I spent a while reading through the forum post when it first went up, but I found it difficult to pick out the few major changes (e.g. Sylph damage increased by 30) from the many minor ones (e.g. Triton cost increased by 3), and I hadn't realized that the main post was missing major changes decided on later-on. Even worse, I hadn't actually looked at the github at all! I had assumed the numbers were identical to the main forum post.

Instead of starting with a forum thread, perhaps it would be simpler to directly make the PRs for each faction, and link to that PR on forums/discord/reddit/etc? That way it's easy to see the up-to-date stats by checking the PR directly.  And discussions might be easier to follow when broken down through Github's conversation system than in a forum thread.

That said, what works best for Hejnewar's workflow is also important, since he's the one actually doing the work.  And frankly, I also do think at least part of this is on the community (including me) for not paying much attention.

**Comment by ForestDragon-wesnoth** [comment]
> I spent a while reading through the forum post when it first went up, but I found it difficult to pick out the few major changes (e.g. Sylph damage increased by 30) from the many minor ones (e.g. Triton cost increased by 3), and I hadn't realized that the main post was missing major changes decided on later-on. Even worse, I hadn't actually looked at the github at all! I had assumed the numbers were identical to the main forum post. It's been a while so my memory is fuzzy, but this is my best recollection.

exactly

**Comment by Dalas121** [comment]
I would also like to mention that while this elf stats issue seems like it came out of nowhere, it's been something I felt strongly about ever since I first noticed the change in-game (because it broke one of my scenarios). But I felt it was unreasonable to complain about an issue that I felt had gone through the community vetting process, and that only I disliked.

Things only came to a head recently after a discussion between multiple people in the UMC server, at which point I realized that a meaningful amount of people felt the same way I did about the change (and presumably they realized the same thing).

**Comment by ForestDragon-wesnoth** [comment]
> But I felt it was unreasonable to complain about an issue that I felt had gone through the community vetting process, and that only I disliked. I have lots of opinions that are just personal preference, in which category I lumped this into.

Many people felt they were alone in disliking it and/or powerless to change this, so they previously stayed quiet and put up with this. But when they see more people voice the same opinion they become less reluctant to join in.
 
> Things only came to a head recently after a discussion between multiple people in the UMC server, at which point I realized that a meaningful amount of people felt the same way I did about the change (and presumably they realized the same thing). There was no plot between ForestDragon and I to simultaneously make issues, we just happened to both feel "_hey something needs to be done_" at the same time.

Yeah, I must admit I was kinda surprised to see you beat me to making a similar issue and noticed yours shortly after publishing mine, but I'm glad to see there's more people who are willing to not just stand aside while such things happen to the game.

**Comment by Pentarctagon** [comment]
@kabachuha Would that just be a link to the forum announcements and another link to the changelog then? If people don't read the announcements in all the places they're posted now, why would they start reading them with this new option?

**Comment by CoxAsh** [comment]
Reading this topic I really have to congratulate the developers for not banning some people. The behaviour here is really not correct from many people, especially because some people came not to comment on the changes but mostly or entirely just attack developers. In my opinion such disruptive behaviour shouldnt be left alone.

**Comment by CelticMinstrel** [comment]
> The elephant in the room is that this team of developers constantly reminds everyone that only devs know and the players feedback will be generally [mocked](https://github.com/wesnoth/wesnoth/issues/6971#issuecomment-1221186649) and discarded (esp. on github). That is the culture enshrined in this project at this time.

What elephant? Wesnoth is an _open_ project. 99% of development work on it is out in the open for _anyone_ to see. If you don't like Pentarctagon, you can vote him out in the next community manager poll. (Though I honestly can't understand why you would want to, as I think he's done a great job… but I suppose it can't hurt to let someone else have a go at the position.)

Of course opinions will be discarded. That goes with the territory. There are thousands of opinions out there, some of them objectively terrible on close inspection, others not necessarily bad but poorly thought out, maybe some that just don't make sense for what Wesnoth is. If exactly one person has a dissenting opinion, and they're not able to convey their argument in a way that convinces others, then yes of course they'll end up being ignored. A project that took _everyone_'s ideas into account to build a great big soup of a game would end up worse for it.

Polls have been mentioned a few times in this thread and the other one, and I've noted that doing them in Discord is unacceptable, but… polls also aren't a great method of decision-making in the first place. We can't make a game out of polls – it just won't become cohesive. Polls can help on particularly divisive or unclear decisions, but decision-making in a game has to be based more on discussion and reaching an eventual consensus rather than polls.

And if some people with a heavy opinion choose not to participate in the discussions, resulting in their opinion going unconsidered, then that's at least partly that person's fault. Sure, maybe Pentarctagon or Hejnewar could've made more of a ruckus about what they were doing and drawn more attention to it, but it was out in the open for anyone to see if they bothered to go looking.

In short, it's not fair to point fingers like that. What's happening right now is basically everyone's fault, and instead of pointing fingers we should just be sitting down and fixing it (which thankfully _is_ happening as well).

**Comment by Mirion147** [comment]
> Reading this topic I really have to congratulate the developers for not banning some people. The behaviour here is really not correct from many people, especially because some people came not to comment on the changes but mostly or entirely just attack developers. In my opinion such disruptive behaviour shouldnt be left alone.

Out of curiosity. Did this comment have value beyond stirring a pot that doesn't need any more stirring? This is a topic that became heated. Joining in after the fact and complaining that some people got heated who opposed the devs opinion does nothing more than irritate those people even more than they are. If you actually want to applaud the devs for being gracious, I suggest doing it in a less heated area.

Let's just try to get back to a place where we can all enjoy a game we clearly feel passionate about.

**Comment by MechanicalDragon963** [comment]
I think just the existence of this issue itself is proof of some miss-communication between the community and developers. This is evidenced by the high hype around the topic from players, developers, and modders. 

I think talking about who is personally attacking whom is futile. Obviously, there is a problem and it needs a solution. One such solution was proposed by ForestDragon - his PR contains a compromise between the old and new characteristics of elves. I think we need to look for such solutions instead of figuring out who is to blame. 

I also think that since the old characteristics of elves were changed, the new ones should not be considered untouchable. Otherwise it's kind of a double standard. I think we need to find a way to increase clarity between the developers and the community, and find a new solution to the balance.

**Comment by demario-w** [comment]
> @demario-w 's text may be excessively harsh

I apologize to the people courageously reporting this issue for the negative light I cast on their report.

I also apologize to the developers who felt personally and unfairly attacked by my words.

For the record I bear no ill will to Pentarctagon (of course!) and I will clarify that I don't know him personally in real life and all my communication to him has been limited to public media.

I thought the PM of this project would take responsibility for the wrongs that happen under their leadership in this project. It is now obvious to me that I was barking at the wrong tree and I will refrain to speak my mind in the future. Best wishes to all.
Good luck to wesnoth as an open source entertainment platform.

**Comment by ProditorMagnus** [comment]
My impression of the overall issue is that mainline balance changes are expected to move slowly, for example few damage points for few units per stable branch giving players time to get used to it.

Reminds me of the time that I imported khthon, then figured out it was massively overpowered and weakened everything, and now for years already it has been getting small buffs.

**Comment by AnekronCZ** [comment]
> My impression of the overall issue is that mainline balance changes are expected to move slowly, for example few damage points for few units per stable branch giving players time to get used to it.

While I agree on principle, I'd defend Hejne on this point. One of the biggest changes he made was rebalancing the arcane damage type entirely, and that's something that will inevitably touch many, many units and the overall balance. I actually think this change was for the better, since especially the new players don't expect such drastic difference in resistance to what's the hardest damage to grasp thematically.
